src/NeuralNetwork/NeuralNetworkData.js

```diff
@@ -754,21 +755,8 @@ class NeuralNetworkData {
    * @param {*} cb
    */
   async saveMeta(nameOrCb, cb) {
-    let modelName;
-    let callback;
-
-    if (typeof nameOrCb === 'function') {
-      modelName = 'model';
-      callback = nameOrCb;
-    } else if (typeof nameOrCb === 'string') {
-      modelName = nameOrCb;
-
-      if (typeof cb === 'function') {
-        callback = cb;
-      }
-    } else {
-      modelName = 'model';
-    }
+    const { string, callback } = handleArguments(nameOrCb, cb);
+    const modelName = string || 'model';
```

Comment:
nonblocking comment:
* `handleArguements()` feels like some kind of super hero magic -- "with great power comes great responsibility" 😂